pages/favourites.js

```diff
@@ -8,23 +8,20 @@ export default function MyPlantsPage({ plants, toggleFavourite }) {
 
   return (
     <>
-      <StyledHeader>My Plants</StyledHeader>
+      <h1>My Plants</h1>
```

Comment:
The `<h1>` could be defined in the global styles and applied to all of the headlines to have consistency.

Reply:
same applies here, we did in styles.js :D